packages/fxa-auth-server/lib/scheme-refresh-token.js

```diff
@@ -15,10 +15,14 @@ const ScopeSet = require('fxa-shared').oauth.scopes;
 // so we limit to the scope below as a safety mechanism
 const ALLOWED_REFRESH_TOKEN_SCHEME_SCOPES = ScopeSet.fromArray(['https://identity.mozilla.com/apps/oldsync']);
 
-module.exports = function schemeRefreshTokenScheme(db, oauthdb) {
+module.exports = function schemeRefreshTokenScheme(config, db, oauthdb) {
   return function schemeRefreshToken(server, options) {
     return {
       async authenticate (request, h) {
+        if (config.oauthDeviceAccessEnabled === false) {
```

Comment:
mmm.... I understand why you added this here, to keep things DRY, but it makes an assumption that the only thing the refreshToken scheme will ever be used for is for OAuth device access. Fine for now, not sure we should make that a long term policy though.

Reply:
Aye. I'll take a followup to remove it on master so we don't live with it forever, ref https://github.com/mozilla/fxa/issues/1585